script/en/validation-inline.php

```diff
@@ -31,6 +19,11 @@
     ],
     'boolean'              => 'This field must be true or false.',
     'confirmed'            => 'The confirmation does not match.',
+    'custom'               => [
```

Comment:
Custom here, it's not good

Reply:
It has been moved alphabetically.

Reply:
I figured out how to keep the location of the `custom` and` attributes` keys at the end of the file. I will do it in the near future.